### PR TITLE
[SMALLFIX] Fixed test in 'NetworkAddressUtilsTest'

### PR DESCRIPTION
**Pull request by HorizonNet**
The ```testGetBindAddress``` method in the ```NetworkAddressUtilsTest``` wasn't actually testing the binding of the addresses. The test is now pointing to the correct helper method. In this method the different service types are now setting the port for their specific configuration.

### CODE REVIEW

**Comment by HorizonNet** [comment]
Jenkins, test this please.

File: common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java [hunk 1]
```diff
@@ -162,7 +162,25 @@ public void getBindAddress(ServiceType service) throws Exception {
...
+        conf.set(Constants.WORKER_RPC_PORT, "20000");
+        break;
+      case WORKER_DATA:
+        conf.set(Constants.WORKER_DATA_PORT, "20000");
+        break;
+      case WORKER_WEB:
+        conf.set(Constants.WORKER_WEB_PORT, "20000");
+        break;
+      default:
+        break;
```
Comment (aaudiber) [inline common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java]: The default case should probably throw an exception

**Comment by aaudiber** [comment]
Thanks for fixing this bug! LGTM overall, just left one comment.

File: common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java (hunk 1 above)
Comment (HorizonNet) [inline common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java]: Do you have a specific one in mind?

Comment (aaudiber) [inline common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java]: Could be `Assert.fail("Unrecognized service type: " + service.toString());`

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comment and the hint. I updated the PR accordingly.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**